packages/api-explorer/src/ResponseSchema.jsx

```diff
@@ -1,78 +1,119 @@
   }
-  selectedStatus(selected) {
+
+  selectedStatus (selected) {
     this.setState({ selectedStatus: selected });
   }
 
-  changeHandler(e) {
-    this.selectedStatus(e.target.value);
+  changeHandler (e) {
+    this.selectedStatus(e.target.value)
   }
 
   // TODO https://github.com/readmeio/api-explorer/issues/43
-  // renderSchema() {
-  // let schema;
-  //
-  // try {
-  //   if (operation.responses[this.state.selectedStatus].content) {
-  //     if (
-  //       operation.responses[this.state.selectedStatus].content['application/json'].schema.type ===
-  //         'object' &&
-  //       operation.responses[this.state.selectedStatus].content['application/json'].schema
-  //         .properties
-  //     ) {
-  //       schema =
-  //         operation.responses[this.state.selectedStatus].content['application/json'].schema
-  //           .properties;
-  //     }
-  //   } else if (
-  //     operation.responses[this.state.selectedStatus].content['application/xml'].schema.type ===
-  //       'object' &&
-  //     operation.responses[this.state.selectedStatus].content['application/xml'].schema.properties
-  //   ) {
-  //     schema =
-  //       operation.responses[this.state.selectedStatus].content['application/xml'].schema
-  //         .properties;
-  //   }
-  // } catch (e) {} // eslint-disable-line no-empty
-
-  /* {schema && (
+  renderSchema (operation) {
+    let schema
+    const oas = this.state.oas
+    if(!this.validateOperation(operation, this.state.selectedStatus)) return
+
+    if (operation.responses[this.state.selectedStatus].content) {
+      if (
+        operation.responses[this.state.selectedStatus].content['application/json'].schema.type ===
+          'object' &&
+        operation.responses[this.state.selectedStatus].content['application/json'].schema
+          .properties
+      ) {
+        schema =
+          operation.responses[this.state.selectedStatus].content['application/json']
+      }
+    }
+    else if (
+      operation.responses[this.state.selectedStatus].content['application/xml'].schema.type ===
+        'object' &&
+      operation.responses[this.state.selectedStatus].content['application/xml'].schema.properties
+    ) {
+      schema =
+        operation.responses[this.state.selectedStatus].content['application/xml']
+    }
+    if(operation.responses[this.state.selectedStatus].content &&
+      operation.responses[this.state.selectedStatus].content['application/json'].schema &&
+      operation.responses[this.state.selectedStatus].content['application/json'].schema.$ref
+    ){
+      schema = {
+        schema: findSchemaDefinition(operation.responses[this.state.selectedStatus].content['application/json'].schema.$ref, oas)
+      }
+    }
+    return schema && (
       <table>
-        {swaggerUtils.convertToParams([response], 'response').forEach(param => {
-          <tr>
-            <th>param.name</th>
+        {this.convertToParams([schema], 'response').map((param, index) => {
```

Comment:
Instead of converting the jsonschema to an array of `params`, can we make it so that we directly work with the JSON schema instead? And iterate over it that way? If we can simplify this code as much as possible (or remove the need for it) that would be great: https://github.com/readmeio/api-explorer/pull/166/files#diff-d04d0d9c6cbb056fe8446853b8a7676eR80